index.php

```diff
@@ -530,6 +513,7 @@
 
 require_once($template);
 
-if (!$a->is_backend())
+if (!$a->is_backend()) {
```

Comment:
The following lines are equivalent to `killme()`, please remplace them by a function call.